repositories/EventRepository.ts

```diff
@@ -57,4 +59,18 @@ export class EventRepository extends BaseRepository<EventModel> {
     }
     return query;
   }
+
+  private getBaseFindEventQuery(): SelectQueryBuilder<EventModel> {
+    return this.repository.createQueryBuilder('event')
+      .leftJoinAndSelect('event.feedback', 'feedback')
+      .leftJoinAndSelect('feedback.user', 'feedback.user');
+  }
+}
+
+@EntityRepository(EventFeedbackModel)
+export class EventFeedbackRepository extends BaseRepository<EventFeedbackModel> {
+  public async addEventFeedback(feedback: string[], event: EventModel, user: UserModel): Promise<EventFeedbackModel[]> {
+    const eventFeedback = feedback.map((comment) => EventFeedbackModel.create({ user, event, comment }));
```

Comment:
I think this way it's easier for frontend to visualize / bucket specific feedback by count to see what's the most common feedback and other stats like that. Plus feedback isn't just emojis, they can be custom (and I imagine they can have commas too) so that's why I stuck to splitting them into several records

Reply:
One tool we have at our disposal is serialization: we can't keep arrays in Postgres but what we can do is serialize the array into a string for storage, so the model has an array of strings that gets serialized into a single string, stored as `text`, and then deserialized as arrays for the app to use. You can see the item option metadata as an example of this, where the model property is a JSON object but it's stored as a string. A set of emoji strings don't really need to be their own models, so that's a trick that's available to us.

Reply:
The default `Array.prototype.toString` is pretty lousy for this since it only delimits using commas, but we can use `JSON.stringify` and `JSON.parse` on arrays to easily keep the strings intact, even if they have commas, quotation marks, or anything else, since they all get escaped.